packages/trader/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -79,14 +78,15 @@ class AccountSignup extends React.Component {
 
         return (
             <div className='account-signup'>
-                <Form
+                <Formik
                     initialValues={ signupInitialValues }
                     validate={ validateSignupPassthrough }
                     onSubmit={ onSignupPassthrough }
                 >
-                    {
-                        ({ isSubmitting, errors, values }) => (
+                    {({ isSubmitting, errors, values, setFieldValue, touched }) => (
+                        <Form>
                             <React.Fragment>
+                                <p>{ JSON.stringify(values, null, 2) }</p>
```

Comment:
what is this for?

Reply:
@oskar-binary My bad. It was for debugging. I will remove it.